packages/edit-site/src/components/navigation-sidebar/navigation-toggle/index.js

```diff
@@ -40,6 +41,15 @@ function NavigationToggle( { icon } ) {
 
 	const disableMotion = useReducedMotion();
 
+	const navigationToggleRef = useRef();
+
+	useEffect( () => {
+		// Focus the trigger button if close
```

Comment:
```suggestion
		// TODO: Remove this effect when alternative solution is merged.
		// See: https://github.com/WordPress/gutenberg/pull/37314
```

Will this to-do commend work, @kevin940726?

Reply:
Perfect! Thanks!